src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/SyntaxTokenToSemanticTokensEditHelper.cs

```diff
@@ -16,84 +16,109 @@ internal static class SyntaxTokenToSemanticTokensEditHelper
             SemanticTokens newTokens,
```

Comment:
Suggest calling this method `ComputeSemanticTokenEdits` to better reflect what it does.